documentation/library_guide/kernel_templates/single_pass_scan.rst

```diff
@@ -7,7 +7,8 @@ inclusive_scan Function Template
 
 The ``inclusive_scan`` function computes the inclusive prefix sum using a given binary operation.
 The function implements a single-pass algorithm, where each input element is read exactly once from
-global memory and each output element is written to exactly once in global memory.
+global memory and each output element is written to exactly once in global memory. This function
+is an implementation of the Decoupled Look-back [#fnote1]_ scan algorithm.
```

Comment:
```suggestion
is an implementation of the decoupled look-back [#fnote1]_ scan algorithm.
```

Reply:
I was trying to be consistent with the radix sort documentation, which capitalizes the word Onesweep.
https://github.com/oneapi-src/oneDPL/blob/a9aabb2b94020634f8f9961471f8af0a2aeb60ea/documentation/library_guide/kernel_templates/esimd/radix_sort.rst?plain=1#L10

Reply:
Aha ok.  Fair enough.